miniapp/io.hpp

```diff
@@ -17,7 +17,7 @@ namespace io {
 
 struct cl_options {
     // Cell parameters:
-    uint32_t cells = 1000;
+    uint32_t cells = 1001;  // The last cell is always a regular spiking neuron that fires once at t=0
```

Comment:
I think it is better if the number of cells in the model does not include the artificial spike source cell.
From a user's point of view, the number of cells is the number of "real" cells.
So I would make the default number of cells 1000, and have the recipe insert the additional spike source cell automatically.